definition-map.js

```diff
@@ -140,10 +140,37 @@ const QUERIES = {
             isPreview: "Requests the preview URL to launch SCORM content. This requires the current user to be in admin or manager role.",
             type: "The type of the Topic Or Course."
         }
+    },
+    CourseById: {
```

Comment:
Looks like some changes got carried over from your other PR -- mind stripping these out?

Reply:
Removed and pushed